contracts/axelar-auth-verifier/src/testutils.rs

```diff
@@ -70,6 +80,11 @@ pub fn generate_signer_set(env: &Env, num_signers: u32) -> TestSignerSet {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
```suggestion
#[cfg(test)]
```
to scope it test visibility. alternatively, move it to test.rs where functions are defined for unit tests only (if you aren't using this method later outside this crate)